eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/io/DbsnpVariantsWriter.java

```diff
@@ -255,23 +262,48 @@ private void writeSubmittedVariantsAndOperations(List<? extends DbsnpVariantsWra
                     .collect(Collectors.toList());
 
             if (mergedIntoList.size() > 1) {
-                throw new IllegalStateException(
-                        "Clustered variant rs" + hashAndAccession.getSecond() + " was merged into several other "
-                                + "clustered variants with the same hash: " + mergedIntoList);
+                DbsnpClusteredVariantEntity activeClusteredVariant = mongoTemplate.findById(
+                        hashAndAccession.getFirst(), DbsnpClusteredVariantEntity.class);
+
+                if (activeClusteredVariant == null || !mergedIntoList.contains(activeClusteredVariant.getAccession())) {
```

Comment:
Is it a plausible scenario to get an active clustered variant with the hash appear after the current batch?

Reply:
yes, but in any case I don't see any problem.
- if the clustered variant that appears later is the 3rd one with the same hash, it will make collision and be handled in the same way as the 2nd.
- if the clustered variant that appears later is the 2nd one, and now we are handling the 1st one, this 1st one won't make collision and will become the active clustered variant for its hash.